installers/go-agent/win/config/wrapper-agent.conf

```diff
@@ -33,10 +31,8 @@ wrapper.java.mainclass=org.tanukisoftware.wrapper.WrapperSimpleApp
 
 # Java Classpath (include wrapper.jar)  Add class path elements as
 #  needed starting from 1
-wrapper.java.classpath.1=lib/wrappertest.jar
-wrapper.java.classpath.2=lib/wrapper.jar
-wrapper.java.classpath.3=agent-bootstrapper.jar
-
+wrapper.java.classpath.1=lib/wrapper.jar
+wrapper.java.classpath.2=agent-bootstrapper.jar
```

Comment:
While you're at it, could you also add `wrapper.ignore_sequence_gaps=true` so as to eliminate the "reservedForFuture" in wrapper-server.conf.

This also gives us the opportunity to index `config/wrapper-properties.conf` from say `100` instead of users having to know what properties to start indexing at.